vtStorAta/AtaCommandExtensionscpp.cpp

```diff
@@ -206,5 +206,32 @@ namespace vtStor
 
             return(Drive->IssueCommand(CommandType, commandDescriptor, Data));
         }
+
+        eErrorCode cAtaCommandExtensions::IssueCommand_ATATrim(std::shared_ptr<IDrive> Drive, U32 CommandType, std::shared_ptr<IBuffer> Data, U16 Count, std::vector<sLBARangeEntry> LBARangeEntryList)
```

Comment:
About LBARangeEntryList, i using vector to contain element (LBA, Sector Count). i call it with "List" key word. I will calculate Count from  LBARangeEntry